packages/code-du-travail-frontend/pages/information/[slug].js

```diff
@@ -58,24 +57,25 @@ const Information = ({
       breadcrumbs,
       contents,
       date,
+      folder,
       description,
       references = [],
       title,
     },
     relatedItems,
   } = { _source: {} },
 }) => {
-  const editoredContent = contents.map(
+  const editorialContent = contents.map(
     ({ type, name, altText, size, html }) => {
-      const reactedContent = processor.processSync(html).result;
+      const reactContent = processor.processSync(html).result;
       return type === "graphic" ? (
         <figure key={name}>
-          <img src={`/static/assets/graphics/${name}.jpg`} alt={altText} />
+          <img src={`/docs/${folder}/graphics/${name}.jpg`} alt={altText} />
```

Comment:
maybe we could copy the name `information` or `editorial_content` and use it instead of `docs` here for consistency ?